recipes/pycraf-gui/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pycraf-gui" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "1403ac72528b2f4600e89d4a1746888162cadc734490d819f2c08c6a4d1a4910" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
I would recommend regenerating the recipe with Grayskull as the `fn:` part is not really needed anymore. Some other things like `--no-deps` can also be removed and will not be included in the Grayskull generated recipe.

I can also review it manually but that seems faster.

Ref: https://github.com/conda-incubator/grayskull

Reply:
Ok, thanks. I'll give that a try.

Reply:
Now, that was quite convenient, I must say. Thanks for the hint. Could you have another look?